Rocket.Chat/Models/Subscription.swift

```diff
@@ -159,4 +159,18 @@ extension Subscription {
         return object
     }
 
+    static func find(name: String, subscriptionType: [SubscriptionType], realm: Realm) -> Subscription? {
+        let predicate = NSPredicate(
+            format: "name == %@ && privateType IN %@",
+            name, subscriptionType.map { $0.rawValue }
+        )
+
+        return realm.objects(Subscription.self).filter(predicate).first
+    }
```

Comment:
If we're only using the method that's using `Realm.shared`, why not only keep that one?

Reply:
I made it in this way because I wanted to keep the pattern with other method with same name: `find(rid: String, realm: Realm)`.
This method is similar with the method that I was writing, and have `realm` as a parameter. Then, to my new method follow it's, I chose to wrote a function that also receive `realm` parameter.

Maybe it is not necessary  and I keep only one method?

Reply:
Yes, let's only keep what's being used. Thanks!